hie-plugin-api/Haskell/Ide/Engine/PluginUtils.hs

```diff
@@ -275,7 +275,7 @@ readVFS :: (MonadIde m, MonadIO m) => Uri -> m (Maybe T.Text)
 readVFS uri = do
   mvf <- getVirtualFile uri
   case mvf of
-    Just (VirtualFile _ txt _) -> return $ Just (Rope.toText txt)
+    Just (VirtualFile _ txt) -> return $ Just (Rope.toText txt)
```

Comment:
We should make `VirtualFile` more abstract and provide an accessor to get the text out (I have this on my dejafu branch).